src/to-be-empty-dom-element.js

```diff
@@ -19,3 +19,16 @@ export function toBeEmptyDOMElement(element) {
     },
   }
 }
+
+/**
+ * Identifies if a element doesn't contain child elements excluded comments
+ * ℹ Node.COMMENT_NODE can't be used because of the following issue 
+ * https://github.com/jsdom/jsdom/issues/2220
+ *
+ * @param {*} element an HtmlElement or SVGElement
+ * @return {*} true if the element only contains comments or none
+ */
+function isEmptyElement(element){
+  const nonCommentChildNodes = [...element.childNodes].filter(node => node.nodeType !== 8);
+  return nonCommentChildNodes.length === 0;
+}
```

Comment:
I wonder why there's no new line at the end of this file. If I recall correctly, prettier should be doing it when you commit. But maybe I am mistaken.

Reply:
I think you are right, but I have problems to commit with the configured pre-commit hook. It always tries to execute a script which can not be found. The simple reason is that the path doesn't contain any slashes.
At the moment I didn't found any solution only to skip the pre-commit hook, I know this is bad, but better than no commit.
If you know a solution I will be really happy.

---------------------------------------------
Operation System: Windows 10
Script File: ~/node_modules/.bin/kcd-scripts.CMD

Reply:
We probably are not properly setup for a smooth dev experience on Windows.